### PR TITLE
bump new rust 1.26

### PR DESCRIPTION
**Pull request by gibix**
Signed-off-by: gibix <gibix@riseup.net>

### CODE REVIEW

**Comment by mpkh** [comment]
@gibix thanks but for users to not rebuild many times there is another task to do: we need to add/adopt this patch for 1.26 https://github.com/Heather/gentoo-gnome/pull/252 e.g. proper multilib support for rust package

**Comment by gibix** [comment]
@mpkh https://github.com/rust-lang/rust/commit/c87a1086dcca15bf7d6d6209b9bfa0893649202e this be supported from 1.26

**Comment by mpkh** [comment]
@gibix it's not about this patch

check ebuild difference: multilib-build was added and more https://www.diffchecker.com/apT64oE1

File: dev-lang/rust/rust-1.26.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		docs = $(toml_usex doc)
+		submodules = false
+		python = "${EPYTHON}"
+		locked-deps = true
+		vendor = true
+		verbose = 2
+		extended = $(toml_usex extended)
+		[install]
+		prefix = "${EPREFIX}/usr"
+		libdir = "$(get_libdir)"
```
[the remark this answers is not in the document]
Comment (o01eg) [reply dev-lang/rust/rust-1.26.0.ebuild]: No, https://github.com/gentoo/gentoo-rust/blob/c962e88b1e717b9b1c421a2397f5639db2fc59ac/dev-lang/rust/rust-9999.ebuild#L128

**Comment by o01eg** [comment]
Also, you can exclude `cargo` from `tools` so it won't conflict with `dev-util/cargo`.

**Comment by mpkh** [comment]
may you correct tabs / spaces :)

**Pull request merged**